### PR TITLE
[bug-fix] Reposition `internal notification sending` feature

### PR DESCRIPTION
**Pull request by JayaShakthi97**
### Purpose

- Reposition `internal notification sending` feature into `Login and Registration` section.
- Remove unnecessary `admin.server.v1` feature.

### Related Issues

- https://github.com/wso2/product-is/issues/21223

### Related PRs

- N/A

### Checklist

- [ ] e2e cypress tests locally verified. (for internal contributers)
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Relevant backend changes deployed and verified
- [ ] [Unit tests](/docs/testing/UNIT_TESTING.md) provided. (Add links if there are any)
- [ ] [Integration tests](https://github.com/wso2/product-is/tree/master/modules/integration) provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines
- [ ] Ran FindSecurityBugs plugin and verified report
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets

### CODE REVIEW

File: features/admin.server-configurations.v1/utils/governance-connector-utils.ts [hunk 1]
```diff
@@ -123,7 +123,8 @@ export class GovernanceConnectorUtils {
                     properties: [
                         "EmailVerification.Enable", // Enable user email verification
                         "EmailVerification.LockOnCreation", // Enable account lock on creation
-                        "EmailVerification.Notification.InternallyManage", // Manage notifications sending internally
+                        // eslint-disable-next-line max-len
+                        ServerConfigurationsConstants.EMAIL_VERIFICATION_NOTIFICATIONS_INTERNALLY_MANAGED, // Manage notifications sending internally
```
Comment (JayaShakthi97) [inline features/admin.server-configurations.v1/utils/governance-connector-utils.ts:127]: Can not fix the max-len warning by splitting into two lines.

**Comment by JayaShakthi97** [comment]
Conflicts are getting due to the releases. Will be resolved once ready to merge.

File: apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2 [hunk 2]
```diff
@@ -271,6 +271,36 @@
...
+                    "{{ operation }}": [
+                        {% for scope in scopes %}
+                            "{{ scope }}"{{ "," if not loop.last }}
+                        {% endfor %}
+                    ]{{ "," if not loop.last }}
+                    {% endfor %}
+                    {% else %}
+                        "create": [],
+                        "read": [],
+                        "update": [],
```
Comment (pavinduLakshan) [inline apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2:298]: Shall we add the "feature" scope as well?

File: features/admin.server-configurations.v1/pages/connector-listing-page.tsx [hunk 3]
```diff
@@ -73,23 +73,35 @@ export const ConnectorListingPage: FunctionComponent<ConnectorListingPageInterfa
...
+    const hasGovernanceConnectorReadPermission: boolean = useRequiredScopes(
+        featureConfig?.governanceConnectors?.scopes?.read
+    );
+    const hasOrganizationDiscoveryReadPermission: boolean = useRequiredScopes(
+        featureConfig?.organizationDiscovery?.scopes?.read
+    );
+    const hasResidentOutboundProvisioningFeaturePermission: boolean = useRequiredScopes(
+        featureConfig?.residentOutboundProvisioning?.scopes?.feature
+    );
+    const hasInternalNotificationSendingReadPermission: boolean = useRequiredScopes(
```
Comment (pavinduLakshan) [inline features/admin.server-configurations.v1/pages/connector-listing-page.tsx:85]: We have used the `has{feature}ViewPermissions` convention for the feature scope check. Better to follow that IMO.

File: apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2 (hunk 2 above)
Comment (JayaShakthi97) [reply apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2:298]: Fixed b4d854d26bb3c124f47304139dd7ebff4bb856c6

**Review by DonOmalVindula — DISMISSED** [review]

**Review by DonOmalVindula — APPROVED** [review]

**Pull request merged**